Tests/Bridge/Symmetric/SymmetricDecryptorTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * This file is part of the Global Trading Technologies Ltd crypt-bundle package.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace Gtt\Bundle\CryptBundle\Tests\Bridge\Symmetric;
+
+use Gtt\Bundle\CryptBundle\Bridge\Symmetric\SymmetricDecryptor;
+use Gtt\Bundle\CryptBundle\Bridge\Symmetric\KeyReader;
+use PHPUnit_Framework_TestCase as TestCase;
+use PHPUnit_Framework_MockObject_MockObject as MockObject;
+
+/**
+ * Tests for symmetric decryptor
+ */
+class SymmetricDecryptorTest extends TestCase
+{
+    /**
+     * Example base64-encoded ciphertext
+     */
+    const BASE64_CIPHERTEXT = 'NdoMAyNHhD/U4QKoPo50ZHym8KbKUok7uikkHTnbHoG/+yAcCTdDbCcs8AwcZU+M3/Hcp7RSUiiJe/gvI0QS3Q==';
+
+    /**
+     * Key reader mock
+     *
+     * @var KeyReader|MockObject
+     */
+    private $keyReader;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
```

Comment:
Need to skip test for PHP_VERISION_ID < 50400 https://travis-ci.org/GlobalTradingTechnologies/crypt-bundle/jobs/118239582#L155-L156